### DISCUSSION TITLE
Not Sure

### DISCUSSION THREAD

**Discussion by brxndonnn** [new92/igfi#25] [Q&A] [answered]
Is this suppose to happen?
![Screenshot 2023-07-07 205129](https://github.com/new92/IGFollowersIncreaser/assets/103594572/17f1f094-6e24-4df3-beee-18f0eec78e55)

**Comment by new92** [comment, marked as answer]
Ok so it's in a way suppose to happen. You may occasionally get an error but all of them are temporary and can be resolved very quickly.
